common/source/docs/common-optitrack.rst

```diff
@@ -74,58 +73,40 @@ Select all markers in Motive and create a rigid body from them. Please refer to
 Configuration the drone
 =======================
 
-- set :ref:`AHRS_EKF_TYPE <AHRS_EKF_TYPE>` to 2 and :ref:`EK2_ENABLE <EK2_ENABLE>` to 1. Currently, external navigation is only supported in EKF2
-- set :ref:`COMPASS_USE <COMPASS_USE>`, :ref:`COMPASS_USE2 <COMPASS_USE2>`, :ref:`COMPASS_USE3 <COMPASS_USE3>` to 0. It makes ardupilot use orientation from motion capture system rather than compass, because there are many source casue electromagnetic interference in indoor enviorment
-- set :ref:`EK2_GPS_TYPE <EK2_GPS_TYPE>` to 3 and :ref:`GPS_TYPE <GPS_TYPE>` to 0 to disable the GPS
-- set :ref:`EK2_POSNE_M_NSE <EK2_POSNE_M_NSE>` to 0.1 or lower to increase the weighting of position measurements from motion capture system.
+- set :ref:`AHRS_EKF_TYPE <AHRS_EKF_TYPE>` to 3 , :ref:`EK3_ENABLE <EK3_ENABLE>` to 1 and :ref:`EK2_ENABLE <EK2_ENABLE>` to 0
+- set :ref:`COMPASS_USE <COMPASS_USE>`, :ref:`COMPASS_USE2 <COMPASS_USE2>`, :ref:`COMPASS_USE3 <COMPASS_USE3>` to 0. It prvents ardupilot from using compass, because there are many source casue electromagnetic interference in indoor enviorment.
```

Comment:
```suggestion
- set :ref:`COMPASS_USE <COMPASS_USE>`, :ref:`COMPASS_USE2 <COMPASS_USE2>`, :ref:`COMPASS_USE3 <COMPASS_USE3>` to 0. It prevents ArduPilot from using compass, because there are many sources causing electromagnetic interference in indoor environment.
```

Reply:
Hi @nkruzan Thank you :)